scenarioo-client/protractor-e2e-scenarioo.conf.js

```diff
@@ -16,6 +16,7 @@
 var PROTRACTOR_BASE_URL = process.env.PROTRACTOR_BASE_URL || 'http://localhost:8500/scenarioo';
 var BRANCH = process.env.BRANCH || 'HEAD';
 var BUILD_NAME = 'build-' + (process.env.BUILD_NUMBER || 'latest');
+var SCENARIOO_TARGET_DIRECTORY = process.env.SCENARIOO_TARGET_DIRECTORY || './scenariooDocumentation';
```

Comment:
ich wär dafür grad nen besseren default zu konfigurieren

Reply:
warum nicht in allen pipelines (Jenkins und Circle CI) gleiches Verzeichnis innerhalb vom build Verzeichnis verwenden für reports??

Variablennamen find ich nicht opimal "SCENARIOO_REPORTS_OUTPUT_DIR" wär etwas klarer

Reply:
Guter Punkt, sehe ich auch so. Wollte noch keine Pfadanpassungen vornehmen, da ich nicht weiß wie es den aktuellen Jenkins betrifft. Aber steht auf meiner Agenda das zu vereinheitlichen.